src/ui_ng/lib/src/create-edit-endpoint/create-edit-endpoint.component.html.ts

```diff
@@ -39,6 +39,10 @@ export const CREATE_EDIT_ENDPOINT_TEMPLATE: string = `
           <input type="password" class="col-md-8" id="destination_password" [disabled]="testOngoing" [readonly]="!editable" [(ngModel)]="target.password" size="20" name="password" #password="ngModel" (focus)="clearPassword($event)">
         </div>
         <div class="form-group">
+          <label for="destination_insecure" class="col-md-4 form-group-label-override">{{'CONFIG.VERIFY_REMOTE_CERT' | translate }}</label>
+          <clr-checkbox #insecure  class="col-md-8" name="insecure" id="destination_insecure" (clrCheckedChange)="insecureChange(insecure.checked)" [(ngModel)]="target.insecure"></clr-checkbox>
```

Comment:
You have two-directions bound the variable 'target.insecure', why do you add event monitor "insecureChange"? I think these are some functions.